2/Dockerfile.localdev

```diff
@@ -46,16 +46,14 @@ LABEL k8s.io.description="Jenkins is a continuous integration server" \
 # 8080 for main web interface, 50000 for slave agents
 EXPOSE 8080 50000
 
-
+RUN rm /etc/yum.repos.d/*
```

Comment:
maybe group this to reduce the number of layers of the image

Reply:
it should be before the copy, we cant group it